server/middleware.go

```diff
@@ -53,9 +56,19 @@ func AuthMiddleware(next http.Handler) http.Handler {
 			http.Error(w, "error: not authorized", http.StatusUnauthorized)
 			return
 		}
-
-		userEntity, ok := storage.Users.GetUserByLogin(claims.Username)
-		if !ok {
+		idle := config.Int("idle")
```

Comment:
Config is already parsed in main.You should do dependency injection and pass config variable here.
The same with db connection. It should be initialized once in main and passed here